spec/bolt/node/winrm_spec.rb

```diff
@@ -275,6 +275,15 @@ def stub_winrm_to_raise(klass, message)
       end
     end
 
+    it "returns a friendly stderr message with ruby.exe missing", winrm: true do
+      with_tempfile_containing('task-rb-winrm', "puts 42", '.rb') do |file|
+        result = winrm._run_task(file.path, 'stdin', {})
+        stderr = result.output.stderr.string
+        expect(stderr).to match(/^Could not find executable 'ruby\.exe'/)
+        expect(stderr).to_not match(/CommandNotFoundException/)
```

Comment:
Added a quick test demonstrating we no longer receive the grungy `CommandNotFoundException` from PowerShell